packages/core/src/lib/markdown-it/plugins/markdown-it-alt-frontmatter.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+// Process "---" => "<frontmatter>
```

Comment:
Can you explain more in this comment, because it's not always going to process `---` to `<frontmatter>`, hence misleading as is

Reply:
SLFR.
I updated it alongside with docs.
It is now ready to be merged.

Reply:
can you add in at least two negative test cases -> `---` in the middle and `---` does not contain correct key val pairs

Reply:
also test is failing, so nope, not ready to be merged

Reply:
Will add testcases.
Some how it is only failing in github actions hmmm